apps/download-service/src/environments/environment.ts

```diff
@@ -26,7 +26,8 @@ const devConfig = {
       process.env.REGULATIONS_ADMIN_URL ?? 'http://localhost:3333/api',
     regulationsApiUrl:
       process.env.REGULATIONS_API_URL ??
-      'https://reglugerdir-api.herokuapp.com/api/v1',
+      'http://localhost:3000/api/v1',
+      //'https://reglugerdir-api.herokuapp.com/api/v1',
```

Comment:
Question: is this the value want for the time being?  ...or was this a mistake?
Shouldn't this be resolved via some env variable magic?

Reply:
Outdated, has been fixed